tokio/tests/coop_budget.rs

```diff
@@ -4,6 +4,8 @@
 use std::sync::atomic::{AtomicUsize, Ordering};
 use std::sync::Arc;
 use tokio::net::UdpSocket;
+use tokio::runtime::coop::has_budget_remaining;
+use tokio::task::consume_budget;
```

Comment:
I want the modules to work out so that this becomes.
```rs
use tokio::task::coop::{consume_budget, has_budget_remaining};
```
I know that `consume_budget` is defined directly in `task`, but we can move it as long as we keep a (possibly hidden) re-export.